packages/client/modules/meeting/components/EditableTemplateName.tsx

```diff
@@ -30,6 +30,7 @@ const EditableTemplateName = (props: Props) => {
   const {name, templateId, teamTemplates, isOwner} = props
   const atmosphere = useAtmosphere()
   const {onError, error, onCompleted, submitMutation, submitting} = useMutationProps()
+  const autoFocus = name === '*New Template' || name.slice(-4) === 'Copy'
```

Comment:
Feels like a brittle way of detecting whether it's a newly created template, but we are doing a similar thing elsewhere: https://github.com/ParabolInc/parabol/blob/f21825c30161d761d49df00a0f6fe2e24ac25792/packages/client/modules/meeting/components/AddNewReflectTemplate.tsx#L80

Reply:
I think it's safe to go forward with this change. I think `name.endsWith(' Copy')` would be nicer to read though.

Reply:
Nice, yeah I prefer `endsWith` 👍